examples/rollbar-spring-webmvc/src/main/resources/application.properties

```diff
@@ -0,0 +1,3 @@
+rollbar.access_token = 1b7dec62e84341ff8361b91d2c94e5b4
```

Comment:
I think probably is better to place here a token placeholder, something like `<ROLLBAR_ACCESS_TOKEN>. Here you are leaking it.

Reply:
done